public/index.html

```diff
@@ -8,10 +8,18 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link rel="apple-touch-icon" sizes="180x180" href="../src/img/favicon/apple-touch-icon.png">
```

Comment:
if you import img like this you don't get it on staging (https://mitalentweb-cduqthroi.now.sh) if you see here you haven't any favicon

you need change href
href="../src/img/favicon/apple-touch-icon.png" -> "%PUBLIC_URL%/favicon/apple-touch-icon.png" 

and in folder `public` add folder `favicon` with all files

folder `src/img/favicon/` delete